HTML-exercise/index.html

```diff
@@ -12,32 +12,33 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
       rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" />
   </head>
 
   <body>
-    <header>
+    <header class="page-header">
       <div class="container">
         <div class="row">
-          <h1 class="header-logo cols-2">
-            <a href="#" class="header-logo-link">
+          <h1 class="header-logo col-2">
+            <a href="#" class="logo-link">
               <img src="asset/images/logo.png" alt="E-shop" />
-              E-Shop
+              <h3 class="shop-name">E-Shop</h3>
```

Comment:
ko cần thêm class bà div ở đây em, về font family design đang ko đúng, em ko cần quan tâm lắm

Reply:
đoạn này em làm đồng bộ với cái logo ở footer rồi, ý tưởng là có thể sử dụng lại được ở nhiều vị trí và nhiều trang khác nhau, chỉ cần thay đổi giá trị thuộc tính về font hay màu sắc  tại một chỗ thì toàn bộ các khối logo này đều sẽ thay đổi theo.

Reply:
ở ngoài dùng h1 rồi em, em thêm thẻ heading ở đây nó làm mất đi ý nghĩa của h1. ko SEO đc.

Reply:
ở ngoài dùng h1 rồi em, em thêm thẻ heading ở đây nó làm mất đi ý nghĩa của h1. ko SEO đc.